packages/app/src/admin/SuperAdminPage.tsx

```diff
@@ -180,7 +191,16 @@ export function SuperAdminPage(): JSX.Element {
           <Button type="submit">Get Database Stats</Button>
         </Stack>
       </Form>
-      <Modal opened={opened} onClose={close} title={modalTitle} centered>
+      <Divider my="lg" />
+      <Title order={2}>Database schema drift</Title>
+      <p>Show the schema migration needed to match the expected database schema.</p>
+      <Form onSubmit={getSchemaDiff}>
+        <Stack>
+          <Button type="submit">Get database schema diff</Button>
```

Comment:
```suggestion
          <Button type="submit">Get Database Schema Diff</Button>
```
Tincey nit...
Same here